main.go

```diff
@@ -3,14 +3,35 @@ package main
 // Multiple packages can be imported together.
 import (
 	"fmt"
-	"math/rand"
-	"time"
+	"os"
+	"bufio"
 )
 
+
+func ReadWords(filename string){
+		//open the text file
+	file, err := os.Open(filename)
+	if err != nil {
+		fmt.Println(err.Error())
+	}
+		//close the file later
+	defer file.Close()
+
+		//read in each file and split on the whitespace
+	scanner := bufio.NewScanner(file)
+	scanner.Split(bufio.ScanBytes)
+	for scanner.Scan() {
+		fmt.Println(scanner.Text())
+	}

```

Comment:
It's not strictly necessary to split on whitespace, but we can. If you do, this function should produce a `string[]`. Otherwise, it should produce a `string`.

Reply:
I'm not splitting on the whitespace, I'm splitting on each byte. What do you mean? Line 22: scanner.Split(bufio.ScanBytes)